partner/partner-management-service/src/main/resources/application-dev.properties

```diff
@@ -61,6 +61,7 @@ pms.cert.service.token.request.clientId=mosip-partner-client
 pmp.ca.certificaticate.upload.rest.uri=https://dev.mosip.net/v1/keymanager/uploadCACertificate
 pmp.partner.certificaticate.upload.rest.uri=https://dev.mosip.net/v1/keymanager/uploadPartnerCertificate
 pmp.partner.certificaticate.get.rest.uri=https://dev.mosip.net/v1/keymanager/getPartnerCertificate/{partnerCertId}
+pmp.partner.original.certificate.get.rest.uri=${mosip.api.internal.url}/v1/keymanager/getPartnerSignedCertificate/{partnerCertId}
```

Comment:
add this in Test folder's properties file also

Reply:
added the properties in test folder